maven_plugin_version-master/pom.xml

```diff
@@ -134,12 +134,12 @@
     <master_plugin_jaxws_version>4.0.3</master_plugin_jaxws_version>
 
     <!-- https://github.com/jakartaee/jax-ws-api -->
-    <master_dependency_jaxws_api_version>3.0.1</master_dependency_jaxws_api_version>
+    <master_dependency_jaxws_api_version>4.0.2</master_dependency_jaxws_api_version>
 
     <!-- Version to generate JAX-WS artifacts -->
     <!-- https://github.com/eclipse-ee4j/metro-jax-ws -->
     <!-- https://github.com/eclipse-ee4j/metro-jax-ws/tree/master/jaxws-ri/extras/jaxws-maven-plugin -->
-    <master_dependency_jaxws_version>3.0.2</master_dependency_jaxws_version>
+    <master_dependency_jaxws_version>4.0.3</master_dependency_jaxws_version>
```

Comment:
4.0.0 seems to be around since Oct 2023, do you know if there is a reason we did not update the dependency the last time?

Reply:
Version 3 is the release for Jakarta EE 9, Version 4 is the release for Jakarta EE 10.
We are switching now from Jakarta EE 9 to Jakarta EE 10.